src/edu/stanford/dlss/was/WasapiResponse.java

```diff
@@ -0,0 +1,83 @@
+package edu.stanford.dlss.was;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class WasapiResponse {
+
+  public WasapiResponse() { }
+
+  public ResponseObject parse(byte[] jsonData) throws IOException {
+    ObjectMapper objectMapper = new ObjectMapper();
+    ResponseObject responseObject = objectMapper.readValue(jsonData, ResponseObject.class);
+    return responseObject;
+  }
+
+
+  /**
+   * Class corresponding to JSON received after making WASAPI request ("FileSet")
+   * @see https://github.com/WASAPI-Community/data-transfer-apis/blob/master/ait-implementation/wasapi/implemented-swagger.yaml#L184-L213
+   */
+  public static class ResponseObject {
+
+    private int count;
+    private String previous;
+    private String next;
+    @JsonProperty("includes-extra")
```

Comment:
Could you do the same thing as you did above with the ordering of annotated properties?